src/side-navigation/side-navigation.scss

```diff
@@ -76,7 +76,7 @@
     height: $iui-baseline * 5;
 
     &,
-    &:hover {
+    &:enabled:hover {
```

Comment:
yeah, I guess we could live with this for some time before we figure out about bigger refactoring..
Have you actually tested it helps?
does not work before your fix and works after...?

Reply:
Well, I could not reproduce the problem at all, because it depends on the order of imports.

Maybe I can change from all.css to individual imports to test locally

Reply:
Yeah, thats what I was asking...

Reply:
Its kinda obvious, what import order is needed to reproduce (its 2 files only and I think this was cought by Vilius test)

Reply:
Oh wow, I tested with different order and found a few more instances where sidenav-button needed extra specificity.

So I added redundant selectors for setting button height and icon size

Reply:
Ah damnit, border styling on focused button is still broken
![image](https://user-images.githubusercontent.com/9084735/121026023-c11cf900-c773-11eb-8c1f-87cd2c3beeb0.png)